lib/modules/dosomething/dosomething_api/resources/reportback_item_resource.inc

```diff
@@ -133,9 +141,19 @@ function _reportback_item_resource_access($op) {
  * @param  bool    $random
  * @param  int     $page
  * @param  bool    $load_user
+ * @param  int     $reaction_user
  * @return array
  */
-function _reportback_item_resource_index($campaigns, $exclude, $status, $count, $random, $page, $load_user) {
+function _reportback_item_resource_index($campaigns, $exclude, $status, $count, $random, $page, $load_user, $reaction_user) {
+  if ($reaction_user) {
+    $reaction_user = user_load($reaction_user);
+
+    if ($reaction_user) {
```

Comment:
I'm sure I'm missing something, but doesn't the `status` parameter on these endpoints get passed to [`removeUnauthorizedStatuses`](https://github.com/DoSomething/phoenix/blob/a05bd197bea695eed6cd0ff58d796d94a886f3a0/lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php#L86-L111) which relies on the global `$user` for the `user_access` check?